src/Discord/Parts/Channel/Channel.php

```diff
@@ -356,6 +430,35 @@ function ($response) use ($deferred) {
         return $deferred->promise();
     }
 
+    /**
+     * Deletes a message.
+     *
+     * @param message|Message or id to delete.
+     *
+     * @return \React\Promise\Promise
+     */
+    public function deleteMessage($message)
```

Comment:
Because of the model-repository architecture, I don't want to add a `delete` function to any parts because it won't update the respective repositories.

Reply:
shouldn't this trigger a MESSAGE_DELETE event on the websocket?